syntax = "proto3";

package qtprotobuf.testrpc;

message EchoRequest {
    string message = 1;
}

message EchoResponse {
    string message = 1;
    repeated EchoResponse list_messages = 2;
}

service EchoService {
    rpc Echo(EchoRequest) returns (EchoResponse);
}
